runtime/darwinia/src/pallets/fee_market.rs

```diff
@@ -12,7 +12,6 @@ frame_support::parameter_types! {
 	pub const TreasuryPalletId: PalletId = PalletId(*b"da/trsry");
 	pub const FeeMarketLockId: LockIdentifier = *b"da/feelf";
 
-	pub const AssignedRelayersNumber: u64 = 3;
```

Comment:
The newly added storage item `AssignedRelayersNumber` 's default value is 3. So nothing to do extra.